run_action.py

```diff
@@ -561,13 +561,20 @@ def main():
         )
     )
 
-    with open(args.clang_tidy_fixes, encoding="utf_8") as file:
-        clang_tidy_fixes = yaml.safe_load(file)
+    if os.path.isfile(args.clang_tidy_fixes):
+        with open(args.clang_tidy_fixes, encoding="utf_8") as file:
+            clang_tidy_fixes = yaml.safe_load(file)
+    else:
+        print(
+            # pylint: disable=line-too-long
+            f"Could not find the clang-tidy fixes file '{args.clang_tidy_fixes}', it is assumed that it was not generated"
+        )
```

Comment:
How about the following to avoid the linter  suppression:

```suggestion
        print(
            f"Could not find the clang-tidy fixes file '{args.clang_tidy_fixes}',"
            "it is assumed that it was not generated"
        )
```

Reply:
@platisd yes, why not.